src/monogame/include/csharp_common.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+typedef char                 cschar;
+typedef unsigned char        csbyte;
+typedef short                csshort;
+typedef unsigned short       csushort;
+typedef int                  csint;
+typedef unsigned int         csuint;
+typedef long long            cslong;
+typedef unsigned long long   csulong;
+typedef bool                 csbool;
```

Comment:
Do you think it is worth making this for example `mgbool` instead of `csbool` ?

Reply:
yes